web/app/styles/app.css

```diff
@@ -200,3 +200,129 @@
 .workflow-panel-cp {
   margin: 10px 10px 10px 10px;
 }
+
+input[type="text"]:disabled {
+  background: #dddddd;
+}
+
+select#soflow {
+  -webkit-appearance: button;
+  -webkit-border-radius: 2px;
+  -webkit-box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.1);
+  -webkit-padding-end: 20px;
+  -webkit-padding-start: 2px;
+  -webkit-user-select: none;
+  background-image: url(http://i62.tinypic.com/15xvbd5.png), -webkit-linear-gradient(#FAFAFA, #F4F4F4 40%, #E5E5E5);
+  background-position: 97% center;
+  background-repeat: no-repeat;
+  border: 1px solid #AAA;
+  color: #555;
+  font-size: inherit;
+  margin-bottom: 15px;
+  margin-top: 5px;
+  overflow: hidden;
+  padding: 3.5px 10px;
```

Comment:
Don't find any reason to use decimal as padding values in px. 
https://stackoverflow.com/questions/7573702/decimal-numbers-for-padding-and-margin